### PR TITLE
feat: [SRM-11219]: adding testcases

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10369)

**Comment by harshilbits** [comment]
retrigger titlecheck

**Review by reetika04 — APPROVED** [review]

**Comment by harshilbits** [comment]
retrigger jest

File: src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts [hunk 1]
```diff
@@ -8,6 +15,6 @@ export enum SLOCondition {
 
 export const sloConditionOptions: SelectOption[] = [
   { label: 'Error Budget remaining percentage', value: SLOCondition.ERROR_BUDGET_REMAINING_PERCENTAGE },
-  { label: 'Error Budget remaining minutes ', value: SLOCondition.ERROR_BUDGET_REMAINING_MINUTES },
-  { label: 'Error Budget Burn Rate is above ', value: SLOCondition.ERROR_BUDGET_BURN_RATE_IS_ABOVE }
+  { label: 'Error Budget remaining minutes', value: SLOCondition.ERROR_BUDGET_REMAINING_MINUTES },
```
Comment (devinder-harness) [inline src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts:18]: text should always come from `strings` file

**Review by devinder-harness — CHANGES_REQUESTED** [review]

File: src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts:18]: Sure , will be making this change in a separate PR. This PR is specifically for testcases.

**Review by devinder-harness — APPROVED** [review]

File: src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/components/Notifications/components/SLONotificationRuleRow/SLONotificationRuleRow.constants.ts:18]: As discussed,  this PR was specifically for testcases. Will make the change in a separate PR as I would have to make change at multiple places.

File: src/modules/85-cv/components/Notifications/components/ConfigureMonitoredServiceAlertConditions/__tests__/ConfigureMonitoredServiceAlertConditions.test.tsx [hunk 2]
```diff
@@ -36,4 +37,25 @@ describe('Unit tests for ConfigureMonitoredServiceAlertConditions', () => {
...
+    const { getByText, container } = render(<WrapperComponent {...props} />)
+
+    expect(getByText('Add Condition')).toBeInTheDocument()
+    userEvent.click(getByText('Add Condition'))
+
+    waitFor(() => expect(container.querySelector(`input[name=conditions.0.condition]`)).not.toBeNull())
+    waitFor(() => expect(container.querySelector(`input[name=conditions.1.condition]`)).not.toBeNull())
+
+    const deleteButton = container.querySelector('[data-name="trash"]')
+    expect(deleteButton).toBeTruthy()
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/Notifications/components/ConfigureMonitoredServiceAlertConditions/__tests__/ConfigureMonitoredServiceAlertConditions.test.tsx:51]: NIT: can we use toBeInTheDocument()

File: src/modules/85-cv/components/Notifications/components/ConfigureSLOAlertConditions/__tests__/ConfigureSLOAlertConditions.test.tsx [hunk 3]
```diff
@@ -36,4 +37,93 @@ describe('Unit tests for ConfigureSLOAlertConditions', () => {
...
+    userEvent.click(typeToSelect)
+    expect(conditionDropdown.value).toBe(sloConditionType)
+
+    //Selecting threshold
+    const valueInputField = container.querySelector('input[name="0.threshold"]') as any
+    expect(valueInputField).toBeInTheDocument()
+    fireEvent.change(valueInputField, {
+      target: { value: '5' }
+    })
+    expect(valueInputField.value).toBe('5')
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/Notifications/components/ConfigureSLOAlertConditions/__tests__/ConfigureSLOAlertConditions.test.tsx:79]: NIT: expect(valueInputField).toHaveValue('5')

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**